lib/landing_session.js

```diff
@@ -202,6 +202,8 @@ class LandingSession extends Session {
 
     if (!this.readyToFinal()) {  // check git rebase/am has been done
       cli.warn('Not yet ready to final');
+      cli.log('A git rebase/am is in progress.' +
+        ' Please complete it before running final');
```

Comment:
s/running final/running `git node land --final`/

Reply:
Changed in 408ed19